[2024-02-04T10:52:54.573+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sf_data_upload.sf311_s3_upload_task manual__2024-02-04T09:55:54.011497+00:00 [queued]>
[2024-02-04T10:52:54.579+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sf_data_upload.sf311_s3_upload_task manual__2024-02-04T09:55:54.011497+00:00 [queued]>
[2024-02-04T10:52:54.580+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-04T10:52:54.588+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): sf311_s3_upload_task> on 2024-02-04 09:55:54.011497+00:00
[2024-02-04T10:52:54.593+0000] {standard_task_runner.py:60} INFO - Started process 941 to run task
[2024-02-04T10:52:54.596+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sf_data_upload', 'sf311_s3_upload_task', 'manual__2024-02-04T09:55:54.011497+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/sf311_data_upload_dag.py', '--cfg-path', '/tmp/tmpmjwiyg1t']
[2024-02-04T10:52:54.597+0000] {standard_task_runner.py:88} INFO - Job 65: Subtask sf311_s3_upload_task
[2024-02-04T10:52:54.630+0000] {task_command.py:423} INFO - Running <TaskInstance: sf_data_upload.sf311_s3_upload_task manual__2024-02-04T09:55:54.011497+00:00 [running]> on host e61977183ded
[2024-02-04T10:52:55.094+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sf_data_upload' AIRFLOW_CTX_TASK_ID='sf311_s3_upload_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-04T09:55:54.011497+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-04T09:55:54.011497+00:00'
[2024-02-04T10:52:55.476+0000] {base.py:83} INFO - Using connection ID 'aws_default' for task execution.
[2024-02-04T10:52:55.478+0000] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-02-04T10:53:27.771+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-04T10:53:27.791+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sf_data_upload, task_id=sf311_s3_upload_task, execution_date=20240204T095554, start_date=20240204T105254, end_date=20240204T105327
[2024-02-04T10:53:27.978+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-04T10:53:28.005+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
